src/components/ProductCard.tsx

```diff
@@ -186,46 +129,86 @@ const ProductCard: React.FC<IProps> = ({
 
   const isImgSrc = typeof image === "string"
 
+  const DESCRIPTION_STYLES: TextProps = {
+    opacity: 0.8,
+    fontSize: "sm",
+    mb: 2,
+    lineHeight: "140%",
+  }
+
   return (
-    <Card>
-      <ImageWrapper background={background}>
+    <Flex
+      flexDirection="column"
+      justifyContent="space-between"
+      color="text"
+      background="searchBackground"
+      boxShadow="0px 14px 66px rgba(0, 0, 0, 0.07)"
+      borderRadius="base"
+      border="1px"
+      borderColor="lightBorder"
+      textDecoration="none"
+      _hover={{
+        transition: "transform 0.1s",
+        transform: "scale(1.02)",
+      }}
+    >
+      <Center
+        background={bgProp}
+        boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
+        minH="200px"
+      >
         {isImgSrc ? (
           <img src={image} alt={alt} />
         ) : (
-          <Image image={image} alt={alt} objectFit="contain" />
+          <Img
+            as={GatsbyImage}
+            image={image}
+            alt={alt}
+            objectFit="contain"
+            width="100%"
+            alignSelf="center"
+            maxW={{ base: "311px", sm: "372px" }}
+            maxH="257px"
+          />
         )}
-      </ImageWrapper>
-      <Content className="hover">
-        <div>
-          {hasRepoData && (
-            <GitStars gitHubRepo={data.repository} hideStars={hideStars} />
-          )}
-          <Title gitHidden={!hasRepoData}>{name}</Title>
-          <Description>{description}</Description>
-          {note.length > 0 && <Description>Note: {note}</Description>}
-        </div>
-        {children && <Children>{children}</Children>}
-      </Content>
-      <SubjectContainer>
+      </Center>
+      <Flex flexDirection="column" p={6} textAlign="left" height="100%">
+        {hasRepoData && (
+          <GitStars gitHubRepo={data.repository} hideStars={hideStars} />
+        )}
+        <Heading
+          as="h3"
+          fontSize="2xl"
+          fontWeight={600}
+          mt={!hasRepoData ? 8 : 12}
+          mb={3}
+        >
+          {name}
+        </Heading>
+        {description && <Text {...DESCRIPTION_STYLES}>{description}</Text>}
+        {note.length > 0 && <Text {...DESCRIPTION_STYLES}>Note: {note}</Text>}
+        {children && <Box mt={4}>{children}</Box>}
+      </Flex>
+      <HStack mt={5} mb={2} px={6} spacing={3}>
         {subjects &&
           subjects.map((subject, idx) => (
             <SubjectPill key={idx} subject={subject}>
               {subject}
             </SubjectPill>
           ))}
         {hasRepoData &&
-          data.repository.languages.nodes.map(
+          data.repository.language.nodes.map(
```

Comment:
Wondering why this has changed 🤔 

Currently I see an error breaking some pages. These two pages are loading but breaking after a second:
- https://ethereumorgwebsitedev01-tylerapfleddererethereumo08606.gatsbyjs.io/en/upgrades/get-involved
- https://ethereumorgwebsitedev01-tylerapfleddererethereumo08606.gatsbyjs.io/en/developers/local-environment/